rotkehlchen/history/events.py

```diff
@@ -355,8 +354,8 @@ def new_step_cb(state_name: str) -> None:
                 fail_callback=fail_history_cb,
                 new_step_data=(new_step_cb, exchange.name),
             )
-            step = self._increase_progress(step, total_steps)
-            step = step_before + STEPS_PER_CEX  # Make sure remote failures don't throw steps off
+            # each exchange instance executes STEPS_PER_CEX steps for total_steps
```

Comment:
What do you mean by the "for total steps" part. It does not make sense in English. Do you mean something like?

```suggestion
            # each exchange instance executes STEPS_PER_CEX steps out of the total_steps
```

or something completely different?

Reply:
It meant towards the count of total_steps. But yes

Reply:
Writing `towards the count of total_steps` is also understandable.